### PR TITLE
LPS-202607 | master

### PR DESCRIPTION
**Pull request by leilanyaragao**
Jira Issue: https://liferay.atlassian.net/browse/LPS-202607

### CODE REVIEW

**Review by igor-franca — APPROVED** [review]

**Comment by igor-franca** [comment]
Thanks for this contribution @leilanyaragao, great work!

**Comment by igor-franca** [comment]
Hey @alinedoleron and @leilanyaragao, I noticed a difference between the getRoles, getUserAccounts and getTerms methods.

The gerRoles and getUserAccounts methods clear the state of the unselected option. We never clear toTerms state.
I did basic tests with this component and I believe that this will not cause any bugs, but in some cases we will keep values ​​in the states because we are not cleaning these states. 

Do you believe that keeping states clean is important for the application?

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/UserNotificationSettings.tsx [hunk 1]
```diff
@@ -116,6 +116,12 @@ export function UserNotificationSettings({
 		setRolesList([]);
 	};
 
+	const getTerms = async () => {
+		const recipientList = values.recipients as UserNotificationRecipients[];
+
+		setToTerms(recipientList.map(({term}) => term).join());
+	};
+
```
Comment (alinedoleron) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/UserNotificationSettings.tsx:124]: Hi @leilanyaragao, please, could you place `getTerms()` function before `getUserAccounts()`? Thanks! 🌷

**Review by alinedoleron — CHANGES_REQUESTED** [review]
Hello, @leilanyaragao. I have just one comment.

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
> Hey @alinedoleron and @leilanyaragao, I noticed a difference between the getRoles, getUserAccounts and getTerms methods.
> 
> The gerRoles and getUserAccounts methods clear the state of the unselected option. We never clear toTerms state. I did basic tests with this component and I believe that this will not cause any bugs, but in some cases we will keep values ​​in the states because we are not cleaning these states.
> 
> Do you believe that keeping states clean is important for the application?

Hi Igor, good point. But I've also tested the pr and I think we can move on without cleaning the states since it's not breaking anything. Thanks!

**Comment by alinedoleron** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**